client/components/social-buttons/google.js

```diff
@@ -61,7 +62,16 @@ class GoogleLoginButton extends Component {
 
 	componentDidMount() {
 		if ( config.isEnabled( 'migration/sign-in-with-google' ) ) {
+			if ( this.props.authCodeFromRedirect ) {
+				this.handleAuthorizationCode( {
+					auth_code: this.props.authCodeFromRedirect,
+					redirect_uri: this.props.redirectUri,
+				} );
+				return;
```

Comment:
I know this is my code, but `return`ing here is problematic: it it fails, the user won't have the opportunity to try again, so we must remove it.

```suggestion
```

Reply:
Removed ! 💪